ios/sdk/src/Info.plist

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>LSApplicationQueriesSchemes</key>
+    <array>
+        <string>dbapi-8-emm</string>
+        <string>dbapi-2</string>
+    </array>
```

Comment:
@hristoterezov, the indentation on GitHub appear incorrect. I think the Info.plist uses tab chars. Anyway, please check and make sure that indentation is correct please (in case it isn't).